pages/knowledge/colours-and-contrast/user-interface-components/README.md

```diff
@@ -0,0 +1,52 @@
+
+## Example: form controls
+
+The following screenshot shows a very simple contact form
+
+![Low contrast form](_media/low-contrast-form.png)
+
+The borders of the input fields and the background colour of the “Send message” button have an insufficient contrast ratio of `1.5:1`. 
+
+Because of that, certain users may have a hard time finding the input fields and enter information. 
+
+They might also struggle to identify the button by its typical shape. However, the button label adds a lot of valuable information, making this case less urgent or even negligible: it has sufficient contrast, it offers a clear invitation to interact by using a verb (which is better than a generic “OK”) and its visual appearance (font, size) is different from the other text elements surrounding it.
+
+## Example: tablists
+
+Tablists convey visually that there is a list of items, and that the content for the currently active item is displayed below. The button-like appearance of the tabs indicates that a user can select them to display their associated content.
+
+The borders of the following tablist have insufficient contrast (`1.5:1`). Users who cannot see them (because they have a visual impairment, because their screen is exposed to direct sunlight, because the beamer has bad contrast settings, etc.) might not realise how the different pieces relate to each other and that they can interact with them.
+
+![Low contrast tablist](_media/low-contrast-tablist.png)
+
+Sometimes, tablists rely on background colours instead of borders to indicate their state and functionality. Although this may look much more obvious, it doesn’t solve the problem because the contrast is still the same.
+
+![Low contrast tablist](_media/low-contrast-tablist-with-background.png)
+
+On the other hand, borders with high contrast make the tablist recognisable for all users:
+
+![Low contrast tablist](_media/tablist-with-borders.png)
+
+And while we are at it: providing clear indicators for where an element starts and where it ends is always good style, so please also expand borders around the content area.
+
+![Low contrast tablist](_media/tablist-with-more-borders.png)
+
+## Exceptions
+
+- **Text:** For text in UI components (for example labels) the same requirements apply as for text in content *unless* the whole component is disabled. (see [chapter about colour contrast for text](/knowledge/colours-and-contrast/text/))
+- **Disabled components:** this could be a disabled text field or button, but also a custom control.
+  - Be careful with this one: often, the mere existence of an element conveys something to a user. For example, a form’s submit button can be disabled while the user’s input is missing (or invalid). Someone who cannot perceive the button in such a moment might draw the wrong conclusions about the form, for example assuming that it can’t be submitted at all.
```

Comment:
I'm unsure whether this shortened explanation is easy enough to understand.

Original text:

> Be careful with components that have low contrast due to having "disabled" status: often, the mere existence of an element conveys something to a user. For example, if a form's submit button is disabled while the user's input is missing (or invalid), and the disabled input (both text and shape) has low contrast, some users may not see the button at all. This could lead them to skip the form, as they may think that it is not to be submitted at all.

Reply:
Emphasize that text *and* shape could have low contrast, making the button disappear for certain people.

Reply:
New text

> Be careful with this one: often, the mere existence of an element conveys something to a user. For example, a form's submit button can be disabled while the user's input is missing (or invalid). If the label and the shape of the button have insufficient contrast, some people might not be able to perceive the button at all. This can mislead them to draw the wrong conclusions about the form, for example assuming that it can't be submitted at all.